common/archiver/filestore/historyArchiver_test.go

```diff
@@ -541,12 +542,13 @@ func (s *historyArchiverSuite) newTestHistoryArchiver(historyIterator archiver.H
 }
 
 func (s *historyArchiverSuite) setupHistoryDirectory() {
+	now := time.Date(2020, 8, 22, 1, 2, 3, 4, time.UTC)
```

Comment:
`time.Now()` returns time with monotonic clock part and it doesn't work for deep comparison (which is used in these tests). I am using some random value instead.